include/dmlc/data.h

```diff
@@ -66,11 +68,11 @@ class DataIter {
  * \brief one row of training instance
  * \tparam IndexType type of index
  */
-template<typename IndexType>
+template<typename IndexType, typename DType = real_t>
```

Comment:
Can we add Documentation for DType, also mention that label and value use the same DType.

Reply:
Done